holiholic_follow/src/main/java/com/holiholic/follow/constant/Constants.java

```diff
@@ -4,8 +4,8 @@
  *
  */
 public class Constants {
-    public final static String GET_TOPICS_URL = "http://localhost:8093/getTopics?md5Key=";
-    public final static String UPDATE_TOPICS_URL = "http://localhost:8093/updateTopics";
-    public final static String GET_PEOPLE_URL = "http://localhost:8093/getPeople?md5Key=";
-    public final static String UPDATE_PEOPLE_URL = "http://localhost:8093/updatePeople";
+    public final static String GET_TOPICS_URL = "http://localhost:8092/getTopics?md5Key=";
```

Comment:
Why 8092, isn't the login module running at this port?

Reply:
follow module runs on port `8093` but this url is used to connect to the database which runs on port `8092`

Reply:
Ah, ok 👍